### PR TITLE
Fix bug: The Content-Type should use the produces in the spec.

### PR DESCRIPTION
**Pull request by xingwu1**
Pick up the first one produce in the list, and use it as Content-Type. Also append "utf-8" as charset if not present.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@xingwu1__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (xingwu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by devigned** [comment]
Please update the pull request from the latest in dev.

File: AutoRest/Modelers/Swagger/OperationBuilder.cs [hunk 1]
```diff
@@ -54,6 +54,17 @@ public Method BuildMethod(HttpMethod httpMethod, string url, string methodName,
             };
 
             method.Documentation = _operation.Description;
+            method.ContentType = "application/json";
+            if (this._effectiveProduces != null && this._effectiveProduces.Count > 0)
+            {
+                method.ContentType = this._effectiveProduces[0];
```
Comment (devigned) [inline AutoRest/Modelers/Swagger/OperationBuilder.cs]: Please search for the first one that produces application/json.

**Comment by xingwu1** [comment]
1. Rebase the branch of dev.
2. Pick the first produce starts with "application/json" instead of any produce.

**Pull request merged**